src/gcovr/formats/gcov/read.py

```diff
@@ -190,130 +177,43 @@ def find_datafiles(search_path: str, exclude_dirs: list[re.Pattern[str]]) -> lis
 # Process a single gcov datafile
 #
 def process_gcov_json_data(
-    data_fname: str, covdata: CoverageContainer, options: Options
+    data_source: str, covdata: CoverageContainer, options: Options
```

Comment:
Any reason for renaming the argument?

Reply:
No strong reason. It felt more descriptive to me. Do you want me to change it back to what it was before ?

Reply:
Yes, please. Then the argument and variable names are aligned.